members.html

```diff
@@ -467,7 +555,7 @@ <h3>Alumni</h3>
           <section id="clients" class="clients wow fadeInUp">
```

Comment:
In the clients css class, could you change line 1231 in `assets/css/style.css` to `opacity: 1;`, which just sets the value to 100% visibility. Also, could you delete lines 1235-1255? They are no longer applicable since we removed the carousel.